CONTRIBUTING.md

```diff
@@ -54,12 +54,12 @@ safely edit anything inside the [docs](./docs) directory, adding new pages or
 editing them. You can edit the sidebar by editing
 [website/sidebars.json](./website/sidebars.json).
 
-Then, to render your changes, run docusaurus's local server. To do this:
+Then, to render your changes, run docusaurus's local server. See [Install docusaurus on your machine](https://docusaurus.io/docs/en/installation.html).
 
- 1. [Install docusaurus on your machine](https://docusaurus.io/docs/en/installation.html).
- 2. `cd website`
- 3. `npm start`
- 4. Navigate to http://localhost:3000/twirp.
+1.  `cd website`
+2.  `npm install`
+3.  `npm start`
+4.  Navigate to http://localhost:3000/twirp.
```

Comment:
This is the only part I updated in this document. The way to install docusaurus changed a little bit and the docs no longer apply. Everything else was prettify in my editor "fixing" the markdown